redbot/cogs/alias/alias.py

```diff
@@ -287,22 +288,48 @@ async def _del_global_alias(self, ctx: commands.Context, alias_name: str):
 
     @alias.command(name="list")
     @commands.guild_only()
+    @checks.bot_has_permissions(add_reactions=True)
     async def _list_alias(self, ctx: commands.Context):
         """List the available aliases on this server."""
         guild_aliases = await self._aliases.get_guild_aliases(ctx.guild)
         if not guild_aliases:
             return await ctx.send(_("There are no aliases on this server."))
-        names = [_("Aliases:")] + sorted(["+ " + a.name for a in guild_aliases])
-        await ctx.send(box("\n".join(names), "diff"))
+        names = sorted(["+ " + a.name for a in guild_aliases])
+        message = ""
+        for a in names:
+            message += "{}\n".format(a)
+        temp = list(pagify(message, delims=["\n"], page_length=1850))
+        alias_list = []
+        count = 0
+        for page in temp:
+            count += 1
+            page = (_("Aliases:\n")) + page + ("\n\nPage {}/{}".format(count, len(temp)))
```

Comment:
"Page {}/{}" needs to be put in `_()` for i18n support. Format fields should also have names to help translators (probably not particularly important in this case, but it's better to do it like that anyway for consistency):
```suggestion
            page = (
                _("Aliases:\n")
                + page
                + _("\n\nPage {page}/{total}").format(page=count, total=len(temp))
            )
```